raiden/tests/utils/blockchain.py

```diff
@@ -105,7 +106,11 @@ def geth_create_account(datadir, privkey):
     """
     keyfile_path = os.path.join(datadir, 'keyfile')
     with open(keyfile_path, 'w') as handler:
-        handler.write(hexlify(privkey))
+        handler.write(hexlify(privkey).decode())
+
+    password_path = os.path.join(datadir, 'pw')
+    with open(password_path, 'w') as handler:
```

Comment:
Here too, it's missing the binary option. (although with the passphrase I don't think it will be a problem)